app/posts/register-trainee-teachers/2023-04-06-allowing-providers-to-import-applications-with-conditions-pending.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Allowing providers to import applications with conditions pending
+description: Making it possible to register trainees whose status in Apply has not yet changed to recruited.
+date: 2023-04-06
+related:
+  items:
+    - text: Apply integration - version 1
+      href: /register-trainee-teachers/apply-integration-v1/
+---
+
+The integration between the Manage and Register services automatically imports applications from Manage when they reach recruited status. This happens when the candidate has either:
+
+- accepted an unconditional offer
+- met all the conditions of their offer
+
+We use recruited status to decide which applications to import because it’s the best way to identify someone who is due to start training.
+
+The integration is not active throughout the year. It’s turned on part way through each recruitment cycle, to help providers register trainees who will start training in the upcoming academic year.
```

Comment:
This isn't right. It *is* on throughout the year, but won't start taking new cycle's applications till we tell it to. Older cycle ones still import.

Reply:
I can rewrite it to be more accurate but do users actually see a difference? Won't applications from the previous cycle already have been imported during the previous cycle?